packages/desktop-client/src/components/reports/graphs/cash-flow-spreadsheet.tsx

```diff
@@ -68,26 +68,25 @@ export function cashFlowByDate(
             { date: { $transform: '$month', $lte: end } },
           ],
           'account.offbudget': false,
-          $or: [
-            {
-              'payee.transfer_acct.offbudget': true,
-              'payee.transfer_acct': null,
-            },
-          ],
         });
 
       if (isConcise) {
         return query
-          .groupBy({ $month: '$date' })
+          .groupBy([{ $month: '$date' }, 'payee.transfer_acct'])
           .select([
             { date: { $month: '$date' } },
+            { xfer: 'payee.transfer_acct' },
```

Comment:
We’re not using the `xfer` abbreviation anywhere else at the moment as far as I can tell, is there a different name that you could use?

Reply:
Changed to `isTransfer`. Does that work better?